packages/backend/src/app/database/database-connection.ts

```diff
@@ -132,4 +102,62 @@ export const databaseConnection = new DataSource({
         FolderEntity,
         PieceMetadataEntity,
     ],
-})
+    synchronize: false,
+}
+
+export const databaseConnection =
+    system.get(SystemProp.ENVIRONMENT) === ApEnvironment.DEVELOPMENT
+        ? new DataSource({
+            type: 'sqlite',
+            database: getSQLiteFilePath(),
+            migrationsRun: true,
+            migrationsTransactionMode: 'each',
+            migrations: [
+                InitialSql3Migration1690195839899,
+            ],
+            ...commonProperties,
+        })
+        : new DataSource({
+            type: 'postgres',
+            host,
+            port,
+            username,
+            password,
+            database,
+            migrationsRun: true,
+            migrationsTransactionMode: 'each',
+            ssl: getSslConfig(),
+            migrations: [
+                FlowAndFileProjectId1674788714498,
+                initializeSchema1676238396411,
+                encryptCredentials1676505294811,
+                removeStoreAction1676649852890,
+                billing1677286751592,
+                addVersionToPieceSteps1677521257188,
+                productEmbed1677894800372,
+                addtriggerevents1678621361185,
+                removeCollectionVersion1678492809093,
+                addEventRouting1678382946390,
+                bumpFixPieceVersions1678928503715,
+                migrateSchedule1679014156667,
+                addNotificationsStatus1680563747425,
+                AddInputUiInfo1681107443963,
+                CreateWebhookSimulationSchema1680698259291,
+                RemoveCollections1680986182074,
+                StoreAllPeriods1681019096716,
+                AllowNullableStoreEntryAndTrigger1683040965874,
+                RenameNotifications1683195711242,
+                ListFlowRunsIndices1683199709317,
+                ProjectNotifyStatusNotNull1683458275525,
+                FlowRunPauseMetadata1683552928243,
+                ChangeVariableSyntax1683898241599,
+                PieceMetadata1685537054805,
+                AddProjectIdToPieceMetadata1686090319016,
+                UnifyPieceName1686138629812,
+                AddScheduleOptions1687384796637,
+                AddAuthToPiecesMetadata1688922241747,
+                AddUpdatedByInFlowVersion1689292797727,
+                AddTasksToRun1689351564290,
+            ],
+            ...commonProperties,
+        })
```

Comment:
Before this PR: postgres logic was hardcoded
After this PR: postgres, and sqlite3 are hardcoded
Future suggestion: make ap database independent

BTW, Do you think that maintaining two sets of migrations for different databases is a good idea?